src/core/expr/fexpr.cc

```diff
@@ -579,6 +579,91 @@ DECLARE_METHOD(&PyFExpr::rowsum)
     ->name("rowsum")
     ->docs(doc_rowsum);
 
+
+static const char* doc_sd =
+R"(sd()
+--
+
+Equivalent to :func:`dt.sd(self)`.
+)";
+
+oobj PyFExpr::sd(const XArgs&) {
+  auto sdFn = oobj::import("datatable", "sd");
+  return sdFn.call({this});
+}
+
+DECLARE_METHOD(&PyFExpr::sd)
+    ->name("sd")
+    ->docs(doc_sd);
+
+
+static const char* doc_shift =
+R"(shift()
+--
+
+Equivalent to :func:`dt.shift(self)`.
+)";
+
+oobj PyFExpr::shift(const XArgs&) {
+  auto shiftFn = oobj::import("datatable", "shift");
+  return shiftFn.call({this});
```

Comment:
`dt.shift()` takes optional parameter `n` (the amount of shift), so this method should do the same.

Reply:
Hi @st-pasha haven't had time to look at this due to other constraints ... I dabbled a bit, but couldnt get to include the extra parameter. Could you kindly show me how to do it. Thanks.

Reply:
done